__tests__/integration.test.js

```diff
@@ -37,3 +39,16 @@ describe("/api/topics", () => {
   });
 });
 
+    describe('/api', () => {
+        test('should respond with an object describing all endpoints', () => {
+          return request(app)
+            .get('/api')
+            .expect(200)
+            .expect('Content-Type', 'application/json; charset=utf-8')
+            .then((response) => {
+                console.log(apiEndpointsJSON)
+              expect(response.body).toEqual(expect.any(Object));
```

Comment:
Great job on comparing this with `apiEndpointsJSON` ! However, there isn't too much point in expecting any object if you're going to compare it with the `apiEndpointsJSON` in the same test. It doesn't reveal anything extra to you as a developer